libs/ledger-live-common/src/env.ts

```diff
@@ -160,6 +160,16 @@ const envDefinitions = {
     parser: intParser,
     desc: "bot's default timeout for scanAccounts",
   },
+  CARDANO_API_ENDPOINT: {
+    def: "https://cardano.coin.ledger.com",
+    parser: stringParser,
+    desc: "Cardano API url",
+  },
+  CARDANO_TESTNET_API_ENDPOINT: {
+    def: "",
```

Comment:
I think it would be better to keep a default value here, for now you can leave the direct link. Later we could consider a reverse proxy for the testnet too, but nothing urgent.